src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts

```diff
@@ -931,6 +931,18 @@ describe('DotEditContentComponent', () => {
                 expect(dotContentletEditorService.clear).toHaveBeenCalledTimes(1);
             });
         });
+
+        describe('save', () => {
+            it('should reload contentlet when saved', () => {
+                spyOn(component, 'reload').and.callThrough();
```

Comment:
we don't should mock method in the component that we are testing

Reply:
the `reload` method is been tested already here: https://github.com/dotCMS/core-web/blob/3f59109e70c8a861b96047bbd5c14b2892018929/src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts#L379

so that functionality is not needed to be tested again